From 4bb2e4b6f0d8796a12483f50dc46618a1b6b50b2 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Wed, 25 Feb 2009 14:09:54 -0800
Subject: [PATCH 22/33] Add Quallcom 3G modem information.

It only has 1 port, so no need to specific the specific one.
(bnc#479726)

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 fdi/information/10freedesktop/10-modem.fdi |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/10-modem.fdi b/fdi/information/10freedesktop/10-modem.fdi
index 621632f..bc9cfdd 100644
--- a/fdi/information/10freedesktop/10-modem.fdi
+++ b/fdi/information/10freedesktop/10-modem.fdi
@@ -454,6 +454,14 @@
         </match>
       </match>
 
+      <!-- Qualcomm 3G modem -->
+      <match key="@info.parent:usb.vendor_id" int="0x05c6">
+        <match key="@info.parent:usb.product_id" int="0x9212">
+          <append key="modem.command_sets" type="strlist">GSM-07.07</append>
+          <append key="modem.command_sets" type="strlist">GSM-07.05</append>
+        </match>
+      </match>
+
 <!-- ***************************************************** 
 			PCMCIA cards 
      ***************************************************** --> 
-- 
1.7.1

